fastplotlib/graphics/_base.py

```diff
@@ -116,6 +116,31 @@ def link(
             callback: callable = None,
             bidirectional: bool = False
     ):
+        '''
+        Link two graphics together
```

Comment:
More description, like link this graphic to another graphic upon an `event_type` to change the `feature` of a `target` graphic.